### PR TITLE
Unsubscribed pages

### PR DESCRIPTION
**Pull request by mbouzi**
located at "/unsubscribed-newsletters" and "/unsubscribed-notifications"
based on these mocks https://www.figma.com/file/0saZ6iBgSsn8BQJUP8k5x1Pu/Storj.io?node-id=3500%3A300

### CODE REVIEW

File: layouts/unsubscribed-notifications/single.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {{partial "head" .}}
+</head>
+<body class="unsubscribed-page page">
+	<div class="container-fluid pt-3 pb-1 storj-blu-bg">
+			<div class="container storj-blu-bg">
+			{{ partial "navbar"  .}}
```
Comment (NikolaiYurchenko) [inline layouts/unsubscribed-notifications/single.html]: missing tab

File: layouts/unsubscribed-newsletters/single.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {{partial "head" .}}
+</head>
+<body class="unsubscribed-page page">
+	<div class="container-fluid pt-3 pb-1 storj-blu-bg">
+			<div class="container storj-blu-bg">
```
Comment (NikolaiYurchenko) [inline layouts/unsubscribed-newsletters/single.html]: extra tab

File: layouts/unsubscribed-notifications/single.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+			</div>
+			<div class="container-fluid pt-3 d-xs-block d-lg-none storj-blu-bg"></div>
+	</div>
+	{{with .Site.GetPage "unsubscribedNotifications" }}
+		{{ partial "unsubscribed/unsubscribed" (dict "message" .Params.msg)}}
+	{{end}}
+	<footer>
+			{{partial "footer" . }}
```
Comment (NikolaiYurchenko) [inline layouts/unsubscribed-notifications/single.html]: extra tab

File: layouts/unsubscribed-notifications/single.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {{partial "head" .}}
+</head>
+<body class="unsubscribed-page page">
+	<div class="container-fluid pt-3 pb-1 storj-blu-bg">
+			<div class="container storj-blu-bg">
```
Comment (NikolaiYurchenko) [inline layouts/unsubscribed-notifications/single.html]: extra tab

**Review by NikolaiYurchenko — COMMENTED** [review]
lg but indentation

File: layouts/unsubscribed-notifications/single.html (hunk 4 above)
Comment (NikolaiYurchenko) [inline layouts/unsubscribed-notifications/single.html]: extra tab

**Review by NikolaiYurchenko — COMMENTED** [review]
lg but indentation

**Review by boshevski — APPROVED** [review]
Looks and works great now. @mbouzi you can check my changes if you like.

**Pull request merged**